subt/drone.py

```diff
@@ -141,13 +141,14 @@ def draw(self):
         #line_obj = plt.plot(t, height, '-o', linewidth=2)
         #plt.legend(iter(line_obj), ('altitude', 'alt - bottom', 'alt + top', 'pose3D.z'))
         #plt.xlabel('time (s)')
-        x = [count for count, H, E, A in self.debug_arr2]
+        x = [count/50.0 for count, H, E, A in self.debug_arr2]  # bottom lidar runnig at 50Hz
         a = [A for count, H, E, A in self.debug_arr2]
         e = [E for count, H, E, A in self.debug_arr2]
         h = [H for count, H, E, A in self.debug_arr2]
         plt.plot(x, a)
         plt.plot(x, e)
         plt.plot(x, h)
+        plt.xlabel('sim time (s)')
```

Comment:
It's approximation since the messages can be dropped out on the ROS side but I guess that does not really matter for this much.

Reply:
Yes, it is misleading (the absolute time) - it can give us hint about period of the oscillations but as I was wondering why the mode changes at time 658s then the answer is that it is +/-100 because of initial simulation time offset. When things calms down I would add input also `sim_time_sec` and compensate for it.

Reply:
I am not sure if that is the best solution. I am thinking about a bigger change that would involve adding "time" to a lot of messages we are sending in osgar which would say at which point of time was the data relevant (ie. for a sensor - when was the data measured etc).